### ISSUE TITLE
Consider making 'rabbitmqctl stop_app' output message more precise

### ISSUE THREAD

**Issue by Ayanda-D** [rabbitmq/rabbitmq-server#1043]
Something small here (unless done on purpose?); currently, output message of `rabbitmqctl stop_app`;

```
Stopping node 'rabbit@host' ...
```

In true essence, the node is still running, and we just stopped the application. The following should be more precise;

```
Stopping rabbit application on node 'rabbit@host' ...
```

I guess the same applies to [rabbitmq-cli](https://github.com/rabbitmq/rabbitmq-cli/blob/master/lib/rabbitmq/cli/ctl/commands/stop_app_command.ex#L33).

**Comment by michaelklishin** [comment]
Most RabbitMQ users don't know the difference between stopping an Erlang application and an Erlang VM. So would it really be a net improvement?

**Comment by Ayanda-D** [comment]
True. Usability wise from an non-Erlang perspective wouldn't seem like an improvement, **but**, it does give a true reflection of the current state of their instance. So it depends on what we're prioritising, users who might confuse this, or a true reflection of the server state.

**Comment by gerhard** [comment]
@michaelklishin should this be implemented in 3.6.7 as well? I'm thinking yes.

**Comment by michaelklishin** [comment]
@gerhard sounds good.

**Comment by gerhard** [comment]
Implemented in both v3.6 and v3.7 CLI, making its way through the pipeline.
